v2/car.go

```diff
@@ -94,21 +106,38 @@ func (h Header) WriteTo(w io.Writer) (n int64, err error) {
 		return
 	}
 	n += wn
-	if err = writeUint64To(w, h.CarV1Offset); err != nil {
+	if err = binary.Write(w, binary.LittleEndian, h.CarV1Offset); err != nil {
 		return
 	}
 	n += uint64BytesSize
-	if err = writeUint64To(w, h.CarV1Size); err != nil {
+	if err = binary.Write(w, binary.LittleEndian, h.CarV1Size); err != nil {
 		return
 	}
 	n += uint64BytesSize
-	if err = writeUint64To(w, h.IndexOffset); err != nil {
+	if err = binary.Write(w, binary.LittleEndian, h.IndexOffset); err != nil {
```

Comment:
it didn't occur to me while reviewing the write side, but it will also likely be significantly cheaper to do all this on []byte, since the sizes here are fixed and pretty small. fine to leave as a TODO until you have benchmarks though, as said before.

Reply:
TODO added. thank you for pointing this out